projects/util-linux/project.yaml

```diff
@@ -14,4 +14,4 @@ sanitizers:
   - address
   - undefined
   - memory
-main_repo: 'https://github.com/karelzak/util-linux'
+main_repo: 'https://github.com/util-linux/util-linux'
```

Comment:
I'm not sure why https://github.com/ossf/scorecard keeps saying that `util-linux` isn't fuzzed though. I wonder what else I should add here to let `scorecard` know that it's really fuzzed.

Reply:
What's weird is that according to `scorecard` `selinux` isn't fuzzed either. I'm puzzled

Reply:
I've just opened https://github.com/ossf/scorecard/issues/1231